public/resources/remix/index.js

```diff
@@ -19,14 +19,29 @@
     }
 
     var detailsBar = document.querySelector(".details-bar");
-    
+
     if (!detailsBar) {
       return;
     }
 
     detailsBar.setAttribute("style", "");
     detailsBar.classList.add("mouse-mode");
 
+    detailsBar.querySelector(".details-bar-remix-button").addEventListener("click",function(event){
+      var projectMetaEl = document.head.querySelector("[name=data-remix-projectId]");
+      if(projectMetaEl) {
+        var projectID = projectMetaEl.getAttribute("content") || false;
+        if(projectID) {
+          ga('send', {
```

Comment:
I would also add a check to make sure `ga` is loaded, so we don't crash here because the GA script was slow to load:

```js
if(projectID && typeof ga === "function") {
...
```